lbrynet/lbrynet_daemon/Daemon.py

```diff
@@ -353,12 +426,15 @@ def __init__(self, root, wallet_type=None):
         self.wallet_user = None
         self.wallet_password = None
 
-        self.internet_connection_checker = LoopingCall(self._check_network_connection)
-        self.version_checker = LoopingCall(self._check_remote_versions)
-        self.connection_problem_checker = LoopingCall(self._check_connection_problems)
-        self.pending_claim_checker = LoopingCall(self._check_pending_claims)
-        self.send_heartbeat = LoopingCall(self._send_heartbeat)
-        # self.lbrynet_connection_checker = LoopingCall(self._check_lbrynet_connection)
+        self.looping_call_manager = LoopingCallManager()
+        looping_calls = [
+            ('internet_connection_checker', CheckInternetConnection(self)),
```

Comment:
these strings should also not be strings. again, not sure how to handle this in python